cloudant-client/src/main/java/com/cloudant/client/api/views/ViewResponse.java

```diff
@@ -68,6 +69,15 @@
     <D> List<D> getDocsAs(Class<D> docType);
 
     /**
+     * Gets a map of the document id and error message for all the of the result rows on this page.
+     * For example, if a doc id does not exist the error message will show "not_found".
+     *
+     * @return a map with an entry for key of the doc id and value of error
+     * @since 2.10.0
+     */
+    Map<String, String> getErrors();
```

Comment:
We should just use the Rows for the general view case, sorry if that wasn't clear yesterday e.g.
```
response.getRows().get(0).getError();
```